tests/test_include.py

```diff
@@ -76,6 +76,17 @@ def test_include_file_with_relative_includes(datadir):
     assert sorted(dicts.keys()) == ["proj1", "proj2", "set_variable"]
 
 
+def test_relative_include_without_root(datadir):
+    dicts = obtain_yaml_dicts(str(datadir / "relative_includes_non_root.yml"))
```

Comment:
Did you make sure this test failed? Regardless, it is probably a good idea to change the cwd to some inner directory:

```python
monkeypatch.chdir(datadir / "proj1")
dicts = obtain_yaml_dicts(str(datadir / "relative_includes_non_root.yml"))
```

Reply:
Done!